generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -27,7 +34,10 @@ public void execute() {
             state.getRuntime().getAddressSpaceView().gc();
             state.getRuntime().getAddressSpaceView().invalidateClientCache();
             state.getRuntime().getAddressSpaceView().invalidateServerCaches();
-            System.out.println("CheckpointOperation Completed");
+
+//            TODO: uncomment when verification supports it
+//            String cpStopRecord = String.format("%s, end, %s", shortName, trimAddress);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)